08LessonEight/in-class-project/homework.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+
+    <title>Blog</title>
+    <link type="text/css" rel="stylesheet" href="css/homework-style.css">
+
+</head>
+
+<body>
+
+   <header>
+
+      <h1>Recent From Blog</h1>
+      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris ni.</p>
+
+   </header>
+<!--images with text are in main section -->
+   <main>
+
+      <section>
+
```

Comment:
the image link here is broken and needs the file extension to be in caps (keyboard.JPG)